overlord/ifacestate/udevmonitor/udevmon.go

```diff
@@ -97,11 +104,30 @@ func (m *Monitor) Disconnect() error {
 // handles hotplug events (devices added or removed). It returns immediately.
 // The goroutine must be stopped by calling Stop() method.
 func (m *Monitor) Run() error {
+	// Gather devices from udevadm info output (enumeration on startup).
+	devices, parseErrors, err := hotplug.EnumerateExistingDevices()
```

Comment:
You don't seem to be doing anything with this `err` here. I would have expected it to be the return value of `Run()` since it is the fatal error from above.